Fixes

MikeSiLVO opened the pull request:
List view fix for Movie Set Discart
 Move IMDb logo from behind Character Art in Fullscreen Panel view

MikeSiLVO commented:
I recommend merging Schimi's PR first and then I will update this PR afterwards if it has any merge conflicts. I didn't notice his pending one before I started commiting. The movie set discart is not perfect yet... still need to add some animations. I might not get to that before I take my little break but I will keep an eye out for when you want to merge this and will update it when you are ready even if I am on my little vaca ;) If I do not notice it shoot me a PM on the forums or reply here since I think I get emailed when either of those happens.

MadMikeDoc commented:
Hi Mike,

I've merged schimi's PR as you suggested. What's up with the movieset discart? and thanks again for what you are doing.

Mie.

MikeSiLVO committed:
[Fullscreen Panel] Move imdb logo from behind character art

MikeSiLVO committed:
Fix Movie Set Discart and rounded poster misplacement

MikeSiLVO commented:
Schimi sent me a PM telling me about no movie set art for the rounded covers option. I checked and he was right so I added it as an include and it now works in both rounded and square cover modes. Still missing the slide animations though. I also adjusted the position of the IMDb logo from behind the character art in Fullscreen Panel view, but that is purely subjective so if you do not like it you can either change it or I can remove it from this PR.

MikeSiLVO committed:
Center movie set discart to match new poster positions
